doc/user_guide/introduction.rst

```diff
@@ -0,0 +1,34 @@
+Introduction
+============
+
+Welcome to the Sales Channel Module of Tryton. A sales
```

Comment:
Welcome to the Sales Channel Module Documentation. The Sales Channel module introduces a way to segment order s in Tryton based on their source. Some channel types also have the ability to export stock and catalog information in addition to being able to import orders from external channels.

The core module introduces minimal functionality required for orders to be taken manually from Tryton Sales module for orders placed on offline channels like emails, phones or by your own sales teams.